webapp/src/DataTransferObject/ResultRow.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace App\DataTransferObject;
+
+class ResultRow
+{
+    public function __construct(
+        public readonly string $teamId,
+        public readonly ?int $rank,
+        public readonly string $award,
```

Comment:
Will everyone now always get an award? Or will `null` now be an empty string?

Reply:
We already had an award for everyone, since even the lowest people get 'Honorable'.